apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js

```diff
@@ -409,32 +449,37 @@ function computePointsFromRubric(sourceInput = null) {
           Math.max(Math.round(itemsSum * 100) / 100, Number(form.dataset.rubricMinPoints)),
           Number(replaceAutoPoints ? form.dataset.maxPoints : form.dataset.maxManualPoints) +
             Number(form.dataset.rubricMaxExtraPoints),
-        ) + Number(form.querySelector('input[name="score_manual_adjust_points"]')?.value || 0);
+        ) + Number(form.querySelector('input[name="score_manual_adjust_points"]')?.value ?? 0);
       const manualPoints =
         rubricValue -
         (replaceAutoPoints
-          ? Number(form.querySelector('.js-auto-score-value-input-points').value)
+          ? Number(form.querySelector('.js-auto-score-value-input-points')?.value ?? 0)
           : 0);
 
-      manualInput.value = manualPoints;
+      manualInput.value = `${manualPoints}`;
     }
   });
   updatePointsView(sourceInput);
 }
 
 function enableRubricItemLongTextField(event) {
+  if (!(event.target instanceof HTMLElement)) {
+    return;
+  }
```

Comment:
Nit (last one of these I'll comment on, you get the idea:

```suggestion
  if (!(event.target instanceof HTMLElement)) return;
```